.github/scripts/install-slices/install_slices.py

```diff
@@ -410,6 +411,7 @@ def main() -> None:
         logging.info("No slices will be installed.")
         return
 
+    extra_flag = '--ignore=unstable' if subprocess.check_output(["chisel","version"]).strip().decode().lstrip("v") > "1.2.0" else ''
```

Comment:
Wouldn't this affect a 1.2.1 release as well?

Reply:
yes, but there isn't one, and the "--ignore" code is already in `main`, so whatever the next version is, it will already support this flag